lib/phoenix/live_dashboard/live/color_bar_legend_component.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Phoenix.LiveDashboard.ColorBarLegendComponent do
+  use Phoenix.LiveDashboard.Web, :live_component
+
+  def render(assigns) do
+    height = if assigns[:height], do: assigns[:height], else: 3
+    formatter = if assigns[:formatter], do: assigns[:formatter], else: &format_percent(&1)
```

Comment:
Btw, don't do this in LV, if you do this, then change tracking won't work. :) Access the assigns only inside the ~L""" bit.